src/Compilers/CSharp/Portable/Syntax/SyntaxKindFacts.cs

```diff
@@ -1048,9 +1050,15 @@ public static SyntaxKind GetPreprocessorKeywordKind(string text)
 
         public static IEnumerable<SyntaxKind> GetContextualKeywordKinds()
         {
-            for (int i = (int)SyntaxKind.YieldKeyword; i <= (int)SyntaxKind.WhenKeyword; i++)
+            ushort firstKeywordIdInclusive = (ushort)SyntaxKind.YieldKeyword;
+            ushort lastKeywordIdExclusive = (ushort)SyntaxKind.ElifKeyword;
+
+            for (ushort id = firstKeywordIdInclusive; id < lastKeywordIdExclusive; ++id)
             {
-                yield return (SyntaxKind)i;
+                if (Enum.IsDefined(typeof(SyntaxKind), id))
```

Comment:
But you're going to need to do that anyways.  Adding a contextual keyword is a lot more than just adding it to this enum.  You'll have many functions and areas around the entire solution that will need updating.  For example, you did not update IsContextualKeyword.